src/openloco/map/tile.h

```diff
@@ -61,11 +61,18 @@ namespace openloco::map
             return map_pos(x, y);
         }
     };
-#pragma pack(pop)
 
-    uint32_t tile_element_height(int16_t x, int16_t y);
+    struct tileHeight
+    {
+        coord_t landHeight;
+        coord_t waterHeight;
+    };
+#pragma pack(pop)
```

Comment:
The size of the struct shouldn't matter. We are not performing memory I/O with it and we aren't passing it to the existing code.